File: python/8 kyu/373.py
# https://www.codewars.com/kata/56cd44e1aa4ac7879200010b/train/python

# Passed

def is_uppercase(inp):
    result = all(not char.islower() for char in inp)
    return result

output = is_uppercase("HELLO I AM DONALD")
print(output)

File: nodejs/7 kyu/373.js
// https://www.codewars.com/kata/586566b773bd9cbe2b000013/train/javascript

// Passed

function noRepeat(str) {
  let charToCountMap = new Object();
  for (const char of str) {
    if (!(char in charToCountMap)) {
      charToCountMap[char] = 0;
    }

    charToCountMap[char]++;
  }

  let result;
  for (const char of str) {
    if (charToCountMap[char] === 1) {
      result = char;
      break;
    }
  }

  return result;
}

const output = noRepeat("aabbccdde");
console.log(output);